doc/ale-java.txt

```diff
@@ -117,6 +117,11 @@ After build completes the files required to run the language server will be
 located inside the folder |org.eclipse.jdt.ls.product/target/repository|.
 Ensure to set |g:ale_java_eclipselsp_path| to the absolute path of that folder.
 
+You could customize compiler options and code assists of the server.
+Under your project folder, modify the file 
+|.settings/org.eclipse.jdt.core.prefs| with options presented at
```

Comment:
Use backticks for this, and remove the pipes around the URL. The pipes are for Vim documentation references.